File: clients/rust/src/my_strategy.rs
use crate::debug_interface::DebugInterface;
use komus24::*;

pub struct MyStrategy {}

impl MyStrategy {
    pub fn new(constants: model::Constants) -> Self {
        Self {}
    }
    pub fn get_order(
        &mut self,
        player_view: &model::PlayerView,
        debug_interface: Option<&mut DebugInterface>,
    ) -> model::Order {
        komus24::model::Order {
            vehicles: vec![],}
    }
    pub fn debug_update(
        &mut self,
        displayed_tick: i32,
        debug_interface: &mut DebugInterface,
    ) {}
    pub fn finish(&mut self) {}
}

File: clients/fsharp/MyStrategy.fs
namespace Komus24

type MyStrategy(constants: Model.Constants) =
    member this.getOrder(playerView: Model.PlayerView, debugInterface: option<DebugInterface>): Model.Order =
        {
            Vehicles = Array.empty
        }

    member this.debugUpdate(displayedTick: int, debugInterface: DebugInterface) = ()

    member this.finish() = ()